src/Chapter1.hs

```diff
@@ -503,7 +504,11 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y =
+  if abs x > (abs y) then
+    y
+  else
+    x
```

Comment:
`()` around second `abs` are redundant. And you also can write it in a single line for brevity. But that's fine to leave as it is :slightly_smiling_face: